packages/commonwealth/client/scripts/views/pages/RewardsPage/utils.ts

```diff
@@ -0,0 +1,52 @@
+export enum MobileTabType {
+  Referrals = 'Refferal Earnings',
```

Comment:
typo

Reply:
thanks, actually i had more typos so went through all of them